TypeCobol/Tools/Options-Config/TypeCobolConfiguration.cs

```diff
@@ -368,12 +375,47 @@ public static OptionSet GetCommonTypeCobolOptions(TypeCobolConfiguration typeCob
             if (!CanCreateFile(config.LogFile) && !string.IsNullOrEmpty(config.LogFile))
                 errorStack.Add(ReturnCode.LogFileError, TypeCobolConfiguration.ErrorMessages[ReturnCode.LogFileError]);
 
+            //CPY Map File
+            try {
+                SetCpyCopiesFile(config.CPYMapFilePath);
+            } catch(Exception e) {
+                //Fail to read the Copy File Name, Log
+                System.IO.File.AppendAllText(config.LogFile != null ? config.LogFile : TypeCobolConfiguration.DefaultLogFileName, e.ToString());
+                if (config.CPYMapFilePath != null)
+                {
+                    errorStack.Add(ReturnCode.CopyNameMapFileError, TypeCobolConfiguration.ErrorMessages[ReturnCode.CopyNameMapFileError]);
+                }
```

Comment:
`VerifFiles` is already there to check external files.

Reply:
But here it is not only for existence, but for any kind of exception while accessing or reading  the file.